app/templates/nodes.html

```diff
@@ -23,7 +23,7 @@ <h1> {{header}} </h1>
 <h1> Nodes </h1>
 {% endif %}
 {% for node in nodes %}
-<a href="{{node.url}}">{{node.wikilink}}</a> ({{node.size()}})<br />
+<a href="{{node.url}}/uprank/foo,bar">{{node.wikilink}}</a> ({{node.size()}})<br />
```

Comment:
I don't get the /foo,bar here, is it a remnant from a test perhaps? :)

Reply:
yeah that's cruft, i'll fix

Reply:
ok updated